### DISCUSSION TITLE
[Concern] Does this allow malicious users to block Udon Security measures?

### DISCUSSION THREAD

**Discussion by Bluscream** [AdGoBye/AdGoBye#27] [Q&A]
Some worlds have udon banlists, or udon admin panels, how are you (planning on) preventing users with malicious intents to misuse this tool.

**Comment by regalialong** [comment]
***I don't intend to block the user and <ins>I cannot.</ins>*** None of the official blocklists block any admin panels and as long as I'm on the AGB Team, I'm going to block anything like that getting upstream, AGB has always been made with good faith usage in mind first and foremost.

I could add some form of detection if something is an known admin panel and prevent the user from doing that, but they can simply recompile the source code with the check removed. The only mitigation is closing down the source of the project and trying to obfuscate builds.

That's a uphill battle I'm predetermined to lose which I don't want to play, ultimately AGB is a *tool* that allows users to control their own experience. I'd like to point out that AGB doesn't do a lot of particularly unique things, anyone could recreate this project with AssetsTools.

I don't know how panels behave when they're disabled on the user locally and I can't test it because I don't own the asset for one.

**Reply by Bluscream** [reply]
Thanks for sharing your thoughts on the subject, thats basically all i wanted to ask regarding this.
